src/foam/nanos/crunch/UserCapabilityJunctionDAO.js

```diff
@@ -298,8 +356,9 @@ foam.CLASS({
           } catch(Exception e) {
             return false;
           }
+          return true;
         }
-        return true;
+        return false;
       `
     },
     {
```

Comment:
ln 365 - I think we might want to invalidate UCJ's that have a one of their perquisites junctions removed.

Reply:
discussed in crunch meeting earlier. 
we should prevent removes altogether and instead use the status to denote validity of the ucj. Will add a rule to invalidate dependent ucj's once a prerequisite ucj falls out of GRANTED status.

Reply:
Yup that works.